code/datums/gamemode/factions/bloodcult/bloodcult_buildings.dm

```diff
@@ -1311,12 +1311,15 @@ var/list/bloodstone_list = list()
 
 /obj/structure/cult/bloodstone/proc/summon_backup()
 	var/list/possible_floors = list()
-	for (var/turf/simulated/floor/F in orange(1,src))
-		possible_floors.Add(F)
+	for (var/turf/simulated/floor/F in range(1,src))
+		if (F != loc)//orange ain't working for some reason
```

Comment:
`orange` excludes the thing you pass as the center and its contents, so in this case it only excludes the bloodstone. You'll want to use `orange(1, get_turf(src))` or similar.